tools/f18/CMakeLists.txt

```diff
@@ -12,11 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if(LINK_WITH_FIR)
```

Comment:
`LINK_WITH_FIR` is in the top-level cmake file too, so that can be eliminated.

Reply:
I left it as past experience leads me to believe no one will want to build/use the new middle end for a while.